### ISSUE TITLE
Exploitation Types - Main Screen - VoiceOver Does Not Read All Exploitation Types for a Country

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#91]
VoiceOver is not reading all the exploited goods in the list for a country when it has more than will fit visually on the screen. Example countries would be Argentina, Bangladesh, Bolivia, and Brazil when all types of exploitations are picked.

**Comment by carolynhuang** [comment]
This appears to be a limitation with the technology and having a scrolling feature - Voiceover will only read what is on the screen.  The scrolling feature was designed so that each country would fall on a separate line.  Do you have a solution for this?  Otherwise, this cannot be resolved without changing the scrolling feature.

**Comment by jfc3-dol** [comment]
We should at least look at a better way to display the content so all the user can see it.

Maybe there is a way to tell screen reader user that there is more data so they can scroll and then continue reading the information. It might be the best way, but it would help. Part of the of how many items show on the screen is the size of the device and if a person is in portrait or landscape mode.

**Comment by sanganalakshay** [comment]
A commit addressing this issue has been pushed on 6th November

**Comment by jfc3-dol** [comment]
Thanks, for adding the number of goods for each country "Exploitation Types". Making this change is a good temporary fix to allow VoiceOver users know how many goods there are and that they might have to scroll the list to hear all the items.

In the next release, we should see if we can come up with a better way to display the "Exploitation Types" goods so VoiceOver will read them all and not only the ones visible to the screen.

Going to close this issue and open a new one to look into a better way to display the information.
